testing/integration-neo4j/src/test/java/com/neueda/jetbrains/plugin/graphdb/test/integration/neo4j/tests/cypher/inspection/CypherExplainWarningInspectionTest.java

```diff
@@ -26,9 +26,9 @@ public void testDataSourceFile_NoHighlight() {
     }
 
     public void testDataSourceFile_HighlightExplainWarning() {
-        addDataSourceFileAndCheck("MATCH (a)-[r:" +
-                "<warning descr=\"The provided relationship type is not in the database.\">" +
-                "ART</warning>]-(b) RETURN *;");
+//TODO: discover, wtf and how to improve
```

Comment:
What's happening?

Reply:
updated todo to more appropriate and descriptive form